krystal-visualization/src/main/java/com/flipkart/krystal/visualization/models/Node.java

```diff
@@ -0,0 +1,15 @@
+package com.flipkart.krystal.visualization.models;
+
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Node {
```

Comment:
Standardize to the terminology "Vajram" everywhere where user sees it and remove "node" (unless its only for internal use)